filebeat/module/system/syslog/ingest/pipeline.json

```diff
@@ -36,10 +36,18 @@
                     "MMM dd HH:mm:ss",
                     "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZZ"
                 ],
-                {< if .convert_timezone >}"timezone": "{{ event.timezone }}",{< end >}
                 "ignore_failure": true
             }
         },
+        {
+            "date": {
+                "if": "ctx.event.timezone != null",
+                "field": "@timestamp",
+                "formats": ["ISO8601"],
+                "timezone": "{{ event.timezone }}",
+                "on_failure": [{"set": {"field": "error.message", "value": "{{ _ingest.on_failure_message }}"}}]
```

Comment:
This is also something we should start to make a "default" instead of using ignore_failure. I wonder if there is a way to concatenate multiple errors together so all errors of one ingest pipeline would end up in error.message either as 1 string or multiple strings.

Reply:
I will give a try to use append here instead of set.